src/lib/pages/backoffice/DocumentRequest/DocumentRequestDetails/DocumentRequestSteps/DocumentRequestStepsHeader.js

```diff
@@ -55,8 +55,8 @@ export class DocumentRequestStepsHeader extends Component {
             step={1}
             currentStep={currentStep}
             iconName="truck"
-            title="Select provider"
-            description="Purchase or borrow from another library"
+            title="Select workflow"
```

Comment:
Little suggestion, maybe better 
```suggestion
            title="Select type"
```